srcDoc/release_notes.rst

```diff
@@ -51,6 +51,19 @@ Release Notes Master
 What's New in Master
 ~~~~~~~~~~~~~~~~~~~~
 
+Use `UDPv4` + `SHMEM` as the default transport configuration (#80)
```

Comment:
use `and` instead of `+`
and `Shared Memory` instead of `SHMEM`

Reply:
Done